docs/yum-download-rpm.md

```diff
@@ -1,18 +1,32 @@
 # Install Percona Server for MySQL using downloaded RPM packages
 
+You should be aware that when you install packages manually, you must resolve and install any dependencies. This process may involve finding and installing the necessary dependencies before installing the packages. Dependencies are other packages that a package may need to function correctly. For example, a package may rely on a specific library. If that library is not installed, or if the installed library is the wrong version, the package may not work correctly.
+
+Package managers, like `APT` or `YUM` install the dependencies for you.
+
 Download the packages from [Percona Product Downloads](https://www.percona.com/downloads). If needed, [Instructions for the Percona Product Download](download-instructions.md) are available.
 
  The following example downloads *Percona Server for MySQL* 8.0.29-21 release packages for *RHEL* 8.
 
-1. Use `wget` to download the tar file.
+Using `Wget`, the following command downloads a specific version of the Percona Server for Red Hat Enterprise Linux 8 from the Percona website. 
 
 	```{.bash data-prompt="$"}
 	$ wget https://downloads.percona.com/downloads/Percona-Server-LATEST/Percona-Server-8.0.29-21/binary/redhat/8/x86_64/Percona-Server-8.0.29-21-rc59f87d2854-el8-x86_64-bundle.tar
 	```
```

Comment:
This command is broken on preview

Reply:
Do you mean the wget command? It works in Docker.

Reply:
https://percona-server-docs-8-0-pr-296.onrender.com/yum-download-rpm.html